index.js

```diff
@@ -174,6 +174,12 @@ Parser.prototype._flush = function(done) {
     // ==========================
     for (var index in self.translations) {
         var key = self.translations[index];
+
+        // simplify ${dot.separated.variables} into just their tails (${variables})
+        key = key.replace(/\$\{([^}]+)\}/g, function(m, expr) {
```

Comment:
What is the purpose of simplifying these? Also this could be written with a single replace and regex.